docs/content/doc/advanced/logging-documentation.en-us.md

```diff
@@ -388,3 +388,14 @@ func newNewoneLogService() {
 
 You should then add `newOneLogService` to `NewServices()` in
 `modules/setting/setting.go`
+
+## Using `logrotate` instead of built-in log rotation
+
+Gitea includes built-in log rotation mechanism, which should be enough for most deployments. However, if you instead want to use `logrotate` program:
+
+-  Disable built-in log rotation by set `LOG_ROTATE` to false on your `app.ini`.
+-  Install `logrotate`.
+-  Configure `logrotate` to match your deployment requirements, see `man 8 logrotate` for configuration syntax details. In particular, use `copytruncate` option instead of restarting Gitea within `postrotate/endscript` block. Ensure that your configurations apply to all files emitted by Gitea loggers as described on above sections.
```

Comment:
```suggestion
-  Configure `logrotate` to match your deployment requirements, see `man 8 logrotate` for configuration syntax details. In particular, use the `copytruncate` option instead of restarting Gitea within the `postrotate/endscript` block. Ensure that your configurations apply to all files emitted by Gitea loggers as described in the above sections.
```